File: cpp/Disk.cpp
#include <bits/stdc++.h>

using namespace std;

class Disk
{
private:
    string type;
    int capacity, price;

public:
    Disk()
    {
        this->type = "-";
        this->capacity = 0;
        this->price = 0;
    }

    Disk(string type, int capacity, int price)
    {
        this->type = type;
        this->capacity = capacity;
        this->price = price;
    }

    void setType(string type)
    {
        this->type = type;
    }

    void setCapacity(int capacity)
    {
        this->capacity = capacity;
    }

    void setPrice(int price)
    {
        this->price = price;
    }

    string getType()
    {
        return this->type;
    }

    int getCapacity()
    {
        return this->capacity;
    }

    int getPrice()
    {
        return this->price;
    }

    ~Disk(){};
};

File: python/Disk.py
class Disk():

    def __init__(self, type="-", capacity=0, price=0):
        self.__type = type
        self.__capacity = capacity
        self.__price = price

    def setType(self, type):
        self.__type = type

    def setCapacity(self, capacity):
        self.__capacity = capacity

    def setPrice(self, price):
        self.__price = price

    def getType(self):
        return self.__type

    def getCapacity(self):
        return self.__capacity

    def getPrice(self):
        return self.__price
